samples/subsys/fs/nffs/README.rst

```diff
@@ -0,0 +1,44 @@
+.. _nrf52_NFFS_sample:
+
+nrf52 NFFS Read/Write Sample
+############################
+
+This is a simple application demonstrating how to read & write persistence data on 
+flash of the nRF52 SoC, using NFFS file system.
+
+Requirements
+************
+
+This sample has been tested on the Nordic nRF52840-PDK board, but would
+likely also run on the nrf52_pca10040 board.
+
+Building and Running
+********************
+
+This sample can be found under ``$(ZEPHYR_BASE)/samples/boards/nrf52/nffs`` in the
+Zephyr tree.
+
+This example adds the ``$(ZEPHYR_BASE)/ext/fs/nffs`` include directory to the project
+by providing its own ``CMakeLists.txt`` within the sample directory.
+
+The following commands build the application.
+
+.. code-block:: bash
```

Comment:
Use `zephyr-app-commands` instead. See examples of usage in `doc/getting_started/`